packages/shell-api/src/shard.ts

```diff
@@ -710,16 +710,24 @@ export default class Shard extends ShellApiWithMongoClass {
       options,
     });
     await this.shardCollection(ns, key, unique, options);
-    // SERVER-92762: Prevent unequal data distribution by setting
-    // numInitialChunks to 1000.
-    const numInitialChunks =
-      typeof unique === 'object'
-        ? unique.numInitialChunks
-        : options?.numInitialChunks;
-    return await this.reshardCollection(ns, key, {
-      numInitialChunks: numInitialChunks ?? 1000,
+
+    if (typeof unique === 'object') {
+      options = unique;
+    }
+
+    const reshardOptions: Document = {
       forceRedistribution: true,
-    });
+    };
+
+    if (options?.numInitialChunks !== undefined) {
+      reshardOptions.numInitialChunks = options.numInitialChunks;
+    }
+
+    if (options?.collation !== undefined) {
```

Comment:
I'm not 100% for this one, but reading the [docs for reshardCollection](https://www.mongodb.com/docs/manual/reference/command/reshardCollection/), it appears that there are cases where `collation` must be included. We would already be passing it correctly to `shardCollection`, but were previously ignoring it for `reshardCollection`, which would have resulted in the latter failing.